### PR TITLE
Synchronize manual with stubs for PHP 8.4 - part 2

### PR DESCRIPTION
**Pull request by kocsismate**
Apparently, all the affected mysqli and Phar methods were always returning `true`.

### CODE REVIEW

**Review by cmb69 — APPROVED** [review]
Thank you!

File: reference/datetime/dateinterval/createfromdatestring.xml [hunk 1]
```diff
@@ -53,7 +53,15 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   Returns a new <classname>DateInterval</classname> instance on success, &return.falseforfailure;.
+   Returns a new <classname>DateInterval</classname> instance.
+  </para>
```
Comment (Girgias) [inline reference/datetime/dateinterval/createfromdatestring.xml:57]: There is an entity IIRC for procedural return false for failure, which needs to be used.

File: reference/datetime/datetime/modify.xml [hunk 2]
```diff
@@ -42,14 +42,14 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   &date.datetime.return.modifiedobjectorfalseforfailure;
+   &date.datetime.return.modifiedobject;
   </para>
```
Comment (Girgias) [inline reference/datetime/datetime/modify.xml:46]: Ditto procedurally, this still returns false.

File: reference/phar/PharFileInfo/delMetadata.xml [hunk 3]
```diff
@@ -27,8 +27,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   Returns &true; if successful, &false; if the entry had no metadata.
-   As with all functionality that modifies the contents of
+   &return.true.always; As with all functionality that modifies the contents of
    a phar, the <link linkend="ini.phar.readonly">phar.readonly</link> INI variable
```
Comment (Girgias) [inline reference/phar/PharFileInfo/delMetadata.xml:31]:
```suggestion
   &return.true.always;
   As with all functionality that modifies the contents of
   a phar, the <link linkend="ini.phar.readonly">phar.readonly</link> INI variable
```

**Review by Girgias — CHANGES_REQUESTED** [review]

**Review by kamil-tekiela — APPROVED** [review]
The mysqli changes are fine.

File: reference/datetime/dateinterval/createfromdatestring.xml (hunk 1 above)
Comment (kocsismate) [reply reference/datetime/dateinterval/createfromdatestring.xml]: Yes, I found it at last

**Review by Girgias — APPROVED** [review]

**Pull request merged**